unified-runtime/source/adapters/level_zero/v2/command_buffer.cpp

```diff
@@ -47,7 +47,44 @@ ur_result_t ur_exp_command_buffer_handle_t_::finalizeCommandBuffer() {
   isFinalized = true;
   return UR_RESULT_SUCCESS;
 }
+ur_result_t ur_exp_command_buffer_handle_t_::awaitExecution(
+    locked<ur_command_list_manager> &commandList) {
+  std::ignore = commandList;
```

Comment:
I'm not sure we should be introducing more cases of `std::ignore`, see https://github.com/intel/llvm/pull/17219